tests/nnapi/specs/V1_1/pad_quant8_nnfw.mod.py

```diff
@@ -11,10 +11,10 @@
            3, 4,]}
 
 output0 = {i3: # output 0
-           [2, 2, 2, 2,
-            2, 1, 2, 2,
-            2, 3, 4, 2,
-            2, 2, 2, 2]}
+           [0, 0, 0, 0,
+            0, 1, 2, 0,
+            0, 3, 4, 0,
+            0, 0, 0, 0]}
```

Comment:
Please revert this change. (refer https://github.com/Samsung/ONE/pull/3220/files#r458522865)